enigma-runtime-t/src/data/state.rs

```diff
@@ -46,6 +46,12 @@ impl IOInterface<EnclaveError, u8> for ContractState {
         self.json[key] = value.clone();
         Ok(())
     }
+
+    fn remove_key(&mut self, key: &str) {
+        if let Some(ref mut v) = self.json.as_object_mut() {
```

Comment:
are we sure it's always an object?

Reply:
For now we are sure, it is created as follows: let json = serde_json::from_str("{}").unwrap();